### PR TITLE
feat: add dockerhub monitoring feature

### PR DESCRIPTION
**Pull request by GeunjeLEE**
### Category
- [x] New feature
- [ ] Bug fix
- [ ] Improvement
- [ ] Refactor
- [ ] etc

### Description
dockerhub webhook plugin receives [`image push(POST)` requests](https://docs.docker.com/docker-hub/webhooks/).
you can this plugin to integrate dockerhub event to cause an action

### Known issue

### CODE REVIEW

**Comment by jihyungSong** [comment]
Interface definition content ( `api/webhook.py` ) to handle webhook API `init` and `verify` is required.

**Review by jihyungSong — CHANGES_REQUESTED** [review]
Interface definition contents ( api/webhook.py ) to handle webhook API init and verify is required.

File: src/spaceone/monitoring/manager/event_manager.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                "is_private": false,
+                "is_trusted": false,
+                "is_official": false,
+                "owner": "spaceoneadmin",
+                "date_created": 1656296314
+              }
+            }
+        """
+
+        base_info = {
```
Comment (jihyungSong) [inline src/spaceone/monitoring/manager/event_manager.py:47]: Why don't pass raw_data directly to `_generate_data()` method, but created `base_info` value and then passed it.

File: src/spaceone/monitoring/manager/event_manager.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        occurred_at = info['pushed_at']
+        title = repo_name
+
+        return {
+            'event_key': event_key,
+            'event_type': 'ALERT',  # ALERT only, docker hub event doesn't have RECOVERY type
+            'title': title,
+            'description': description,
+            'severity': 'INFO',
+            'rule': rule,
```
Comment (jihyungSong) [inline src/spaceone/monitoring/manager/event_manager.py:91]: I wonder why you set `repository.full_description` in rule .

**Review by jihyungSong — CHANGES_REQUESTED** [review]

File: src/spaceone/monitoring/manager/event_manager.py (hunk 2 above)
Comment (GeunjeLEE) [reply src/spaceone/monitoring/manager/event_manager.py:91]: It's a setting mistake, i will fix it.

**Comment by GeunjeLEE** [comment]
@whdalsrnt @jihyungSong 
Any other feedback?

**Pull request merged**